suricata/update/main.py

```diff
@@ -619,7 +619,9 @@ def dump_sample_configs():
 def resolve_flowbits(rulemap, disabled_rules):
     flowbit_resolver = rule_mod.FlowbitResolver()
     flowbit_enabled = set()
+    round = 1
     while True:
+        logger.debug("Checking flowbits for round %s of rules.", round)
```

Comment:
What we're doing here is generally called "multi-pass". So I think we should use the work pass instead.
```
Checking flowbits for pass %d of rules."
```

Reply:
I have incorporated the suggested change.
But I would like to know a little more about what you mean by multi-pass and work-pass.

Reply:
Oops, that was a type.  I meant to write "word", not "work".